src/main/java/mateusz/michal/chat/Service/RegistrationService.java

```diff
@@ -42,71 +62,23 @@ private User createUser(UserDTO userDTO){
         return user;
     }
 
-    private boolean isNameNotPresent(String name){
-        return name.equals("");
-    }
-
-    private boolean isNameIncorrect(String name){
-        return (name.startsWith(" ") || name.endsWith(" ") || hasNameSpecialCharacters(name));
-    }
-
-    private boolean isEmailNotPresent(String email){
-        return email.equals("");
-    }
-
-    private boolean isPassworNotPresent(String password){
-        return password.equals("");
-    }
-
-    private boolean hasNameSpecialCharacters(String name){
-        for(char c : name.toCharArray()){
-            if(!(Character.isLetterOrDigit(c))){
-                return true;
-            }
-        }
-        return false;
-    }
-
-    private boolean isNameNull(String name){
-        return name == null;
-    }
-
-    private boolean isEmailNull(String email){
-        return email == null;
-    }
-
-    private boolean isPasswordNull(String password){
-        return password == null;
-    }
-
-    private boolean isUserInDatabaseByName(String name){
-        Optional<User> user = Optional.ofNullable(userRepository.findByName(name));
-        return user.isPresent();
-    }
-
-    private boolean isUserInDatabaseByEmail(String email){
-        Optional<User> user = Optional.ofNullable(userRepository.findByEmail(email));
-        return user.isPresent();
-    }
-
-    private boolean isEmailIncorrect(String email){
-        Pattern pattern = Pattern.compile("[a-zA-Z]+[\\.a-zA-Z0-9]*@[a-zA-Z0-9]+\\.[a-z]{2,}[a-z]*");
-        return !pattern.matcher(email).matches();
-    }
-
-    private boolean isPasswordIncorrect(String password) {
-        Pattern pattern = Pattern.compile("(?=.*\\d)(?=.*[a-z])(?=.*[A-Z])" +
-                "(?=.*[\\!\\@\\#\\$\\%\\^\\&\\*])(?!=.*\\s).{8,15}");
-        return !pattern.matcher(password).matches();
+    // method that validate request form
+    private List<MyError> validateRegistrationRequest(UserDTO userDTO){
```

Comment:
nit: pewnie wystarczyłoby `validateRequest` - jako że jesteśmy w `RegistrationService` :)

Reply:
Wystarczyłoby.
W szczególności, że to tylko komentarz.